.unibeautifyrc.yml

```diff
@@ -0,0 +1,19 @@
+---
+_:
+  indent_char: " "
+  indent_size: 2
+TypeScript:
+  indent_char: " "
+  indent_size: 2
+JavaScript:
+  indent_char: " "
+  indent_size: 2
+CSS:
+  indent_char: " "
+  indent_size: 2
+HTML:
+  indent_char: " "
+  indent_size: 2
+JSON:
```

Comment:
Let's remove the `JSON` formatting. It differs from `npm`'s `package.json` file format.

Just delete the whole `JSON` property, including sub-properties. Thanks!

Reply:
Done.